controllers/vspherecluster_reconciler.go

```diff
@@ -223,25 +220,6 @@ func (r *clusterReconciler) reconcileNormal(ctx context.Context, clusterCtx *cap
 
 	clusterCtx.VSphereCluster.Status.Ready = true
 
```

Comment:
There is nothing (anymore) in the Reconcile func that depends on the wl cluster APIserver being online

(this was different in previous versions where this controller deployed CCM/CSI)